hw/ip/i2c/dv/README.md

```diff
@@ -65,27 +61,27 @@ The `i2c_base_vseq` virtual sequence is extended from `cip_base_vseq` and serves
 All test sequences are extended from `i2c_base_vseq`.
 It provides commonly used handles, variables, functions and tasks that the test sequences can simple use / call.
 Some of the most commonly used tasks / functions are as follows:
-* task 1:
-* task 2:
+* `i2c_init`: Setup the control register, identifier and clear the FIFOs.
+* `wait_{host|target}_for_idle`.
+
+There's also another test sequence which multiple other tests use as a base and that is `i2c_rx_tx_vseq`.
```

Comment:
I think this needs to be rephrased, specially the end. Maybe:
```suggestion
There's also the test sequence `i2c_rx_tx_vseq` which multiple other tests use as a base.
```

Reply:
How about...
```suggestion
Another test sequence used as a base for other sequences is `i2c_rx_tx_vseq`.
```

Reply:
Thanks, made this change.